### DISCUSSION TITLE
Détection de lilypond

### DISCUSSION THREAD

**Discussion by MorphinT** [Jean-Romain/tabliato#42] [Q&A] [answered]
Bonjour, je suis sur un ordinateur qui tourne sous windows 11. J'ai téléchargé Tabliato, mais lorsque je veux lancer la compilation, l'application m'indique que lilypond n'est pas installé. Or j'ai télécharger lilypond, et j'ai désarchivé le dossier. Ai-je fais une erreur de manipulation ? Merci d'avance

**Comment by Jean-Romain** [comment, marked as answer]
Bonjour, avant il y avait un installateur Windows. Là, je confirme qu'il s'agit d'un zip sur le site de `lilypond`. Décompresser le zip ne suffit pas à installer le logiciel, c'est-à-dire l'enregistrer dans la base de registre Windows pour que `tabliato` le trouve.

Je pense que ce lien devrait fonctionner https://lilypond.org/doc/v2.23/Documentation/web/windows

**Reply by MorphinT** [reply]
Effectivement ça marche maintenant. Désolé d'avoir créé un fil de discussion pour une question aussi basique.
Merci beaucoup !

**Reply by Jean-Romain** [reply]
Pas de problème, au contraire ça me fait plaisir que quelqu'un s'intéresse au logiciel. Et puis la question n'est pas si basique, le lien envoyait bien vers l'exécutable avant. Je vais changer le lien.
